dev/src/sidebar-config.js

```diff
@@ -22,12 +22,12 @@ export const sidebarConfig = [
   },
   {
     title: 'Forms',
-    startOpen: false,
+    startOpen: true,
     links: [
-      { to: 'input', title: 'Input' },
       { to: 'select', title: 'Select' },
       { to: 'slider', title: 'Slider' },
       { to: 'switch', title: 'Switch' },
+      { to: 'textfield', title: 'Text Field' },
```

Comment:
Sometimes it's `Text Field`, sometimes `TextField`, and sometimes `Textfield`. I guess because of how the element should look (`<w-textfield>`), we should use `Textfield` everywhere here, like in the case of `Textarea`?

Reply:
Yup, I know. 😄 Took me some turns to decide what to use where. 😅  So I decided that the component name is single word textfield... but used in titles and text it would be more correct to separate it as Text Field? 🤷

Reply:
Makes sense. We could do the same in the tech-docs then!